docs/src/docs-theme.scss

```
@@ -80,3 +80,18 @@ footer {
         margin: 0;
     }
 }
+
+.table_row_removed {
```

Minsta möjliga:

```suggestion
.table tbody .table__row
  &.table_row_removed {
```

Jag föreslår att vi också använder en extra klass eller id för att bara välja ut just denna tabellen. Det gör det nog enklare att förstå vad dessa klasser används till.

```diff
-<div class="">
+<div class="support-table">
```

```suggestion
.table.support-table tbody .table__row
  &.table_row_removed {
```